[2013-04-10 05:30:46] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 05:37:11] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 05:44:55] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 05:51:17] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 05:51:54] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:13:04] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Parse: syntax error, unexpected ';', expecting '{' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 3' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:3
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:13:52] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 32' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:32
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:16:26] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:16:50] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_052648_add_codes_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:17:30] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Compile Error: Cannot redeclare class AddCodesToUsersTable in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_053647_add_acivation_code_to_users_table.php line 33' in /home/felipillo/public_html/test1/app/database/migrations/2013_04_10_053647_add_acivation_code_to_users_table.php:33
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:18:57] log.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Error: Class 'AddAcivationCodeToUsersTable' not found in /home/felipillo/public_html/test1/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php line 301' in /home/felipillo/public_html/test1/vendor/laravel/framework/src/Illuminate/Database/Migrations/Migrator.php:301
Stack trace:
#0 /home/felipillo/public_html/test1/bootstrap/compiled.php(4014): Symfony\Component\Debug\ErrorHandler->handleFatal()
#1 [internal function]: Illuminate\Exception\ExceptionServiceProvider->Illuminate\Exception\{closure}()
#2 {main} [] []
[2013-04-10 06:21:34] log.ERROR: exception 'RuntimeException' with message 'Not enough arguments.' in /home/felipillo/public_html/test1/vendor/symfony/console/Symfony/Component/Console/Input/Input.php:76
Stack trace:
#0 /home/felipillo/public_html/test1/vendor/symfony/console/Symfony/Component/Console/Command/Command.php(235): Symfony\Component\Console\Input\Input->validate()
#1 /home/felipillo/public_html/test1/vendor/laravel/framework/src/Illuminate/Console/Command.php(96): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /home/felipillo/public_html/test1/vendor/symfony/console/Symfony/Component/Console/Application.php(940): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /home/felipillo/public_html/test1/vendor/symfony/console/Symfony/Component/Console/Application.php(204): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MakeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 /home/felipillo/public_html/test1/vendor/symfony/console/Symfony/Component/Console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 /home/felipillo/public_html/test1/artisan(59): Symfony\Component\Console\Application->run()
#6 {main} [] []
